src/platforms/ios/ioscontroller.mm

```diff
@@ -160,6 +160,11 @@
       }
       onboardingCompletedCallback:^() {
         MozillaVPN::instance()->onboardingCompleted();
+      }
+      allowVpnConfigOptionSelectedCallback:^() {
```

Comment:
not opposed to better name ideas for this

Reply:
Maybe something like onTunnelConfigurationCompleted(). 

Is it possible to do call connectionManager->startHandshakeTimer(); in the existing onboardingCompletedCallback()?

Reply:
Consider something like tunnelConfigSavedCallback or tunnelConfigCompletedCallback, which matches the existing onboardingCompletedCallback